src/mudlet.cpp

```diff
@@ -1194,6 +1194,31 @@ void mudlet::commitLayoutUpdates()
     }
 }
 
+bool mudlet::setUserWindowFontSize(Host* pHost, const QString& name, int size)
+{
+    if (!pHost) {
+        return false;
+    }
+
+    QMap<QString, TConsole*>& dockWindowConsoleMap = mHostConsoleMap[pHost];
+    QMap<QString, TDockWidget*>& dockWindowMap = mHostDockConsoleMap[pHost];
+
+    if (dockWindowMap.contains(name) && dockWindowConsoleMap.contains(name)) {
+        qDebug() << "Setting UserWindow FontSize to" << size << " for: " << name;
+        TConsole* pC = dockWindowConsoleMap[name];
+        pC->console->mDisplayFont = QFont("Bitstream Vera Sans Mono", size, QFont::Normal);
+        pC->console->updateScreenView();
```

Comment:
What am I doing twice?  I honestly don't know, what or why.  I just copied the working code form the miniconsoles which didn't work initially until I called `setIsMiniConsole();` on both `console` and `console2` which I assume compose the top and bottom parts of a scrolled window.  
When opening a window, if `setUserWindowFontSize()` is not called there seems to be a graphical bug where letters are spaced by around twice what they should be until a font size is set, so I stuck that in there with the Opening code.